README.md

```diff
@@ -304,6 +304,26 @@ Waits for element to show up on the page.
 
 ```
 
+
+### .waitForElementToContain(selector, callback)
+
+Waits for the element to have some content.
+
+```javascript
+'Has the google ad loaded': function(done)
+{
+   // need to wait for all the element to be present
+  this.waitForElement('[data-role="page_first_ad"]', function()
+  {
+    this.waitForElementToContain('[data-role="page_first_ad"]', function() {
+      assert.contains(this.$('[data-role="page_first_ad"]').find('div:first').attr('id'), 'google_ads_iframe_');
+      done();
+    });
+  });
+}
```

Comment:
Here you go :) Seems like unexpected behavior from user's point of view. And spilling implementation details.

Can you add extra check then that makes sure element is there and not empty.

Btw, method name was just a suggestion, feel free to modify it to your liking :)

Thank you.